packages/components/src/hooks/useKeyboardController.native.ts

```
@@ -0,0 +1,4 @@
+export {
+  useReanimatedKeyboardAnimation,
+  useKeyboardState,
+} from './useKeyboardController';
```

ESLint

[error] Insert `⏎`

(prettier/prettier)